cms-common/src/main/java/io/commercetools/sunrise/cms/CmsService.java

```diff
@@ -0,0 +1,31 @@
+package io.commercetools.sunrise.cms;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * Service that provides page content, coming from some sort of Content Management System (CMS).
+ */
+@FunctionalInterface
+public interface CmsService {
+
+     /**
+     * Gets the content corresponding to the given CMS identifier for the first found given language.
+     * @param locales the list of locales used to translate the message
+     * @param cmsIdentifier identifier of the CMS entry field
+     * @return the {@code completionStage} of the content in the first found given language, or absent if it could not be found
+     */
+    CompletionStage<Optional<String>> get(final List<Locale> locales, final CmsIdentifier cmsIdentifier);
```

Comment:
Should use

```
CompletionStage<Optional<String>> get(final Iterable<Locale> locales, final CmsIdentifier cmsIdentifier)
```

to allow also SortedSets?

In Sunrise I found that the list of Locales sometimes contains duplicates.